### ISSUE TITLE
[Data@TAMU] Search research project community page

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#12]
**URL:** https://data-pre.library.tamu.edu/communities/e98002c6-8134-4424-90e5-59b721f6faca/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.3.1 Info and Relationships (A)
**Issues:** Two H1 designations
**Notes:** Both "Research Projects" and "Search Results" are labeled H1
